src/components/accessibility/screen_reader_live/screen_reader_live.tsx

```diff
@@ -62,11 +78,25 @@ export const EuiScreenReaderLive: FunctionComponent<EuiScreenReaderLiveProps> =
      * for more examples of the double region approach.
      */
     <EuiScreenReaderOnly>
-      <div>
-        <div role={role} aria-atomic="true" aria-live={ariaLive}>
+      <div ref={focusRef} tabIndex={focusRegionOnTextChange ? -1 : undefined}>
+        <div
+          role={role}
+          aria-atomic="true"
+          // Setting the aria-hidden attribute prevents VO from announcing route changes
+          // differently depending on which status region currently has text.
+          aria-hidden={toggle ? undefined : 'true'}
+          // Setting the aria-live to "off" when focusRegionOnTextChange prevents double
+          // announcement by VO + Firefox on MacOS, likely other pairings.
+          aria-live={focusRegionOnTextChange ? 'off' : ariaLive}
```

Comment:
I did some local testing and was hoping `aria-hidden` would mean we didn't have to set a conditional `aria-live`, but it looks like we do need both. Could I suggest consolidating the comments in that case?
```suggestion
          // Setting `aria-hidden` and setting `aria-live` to "off" prevents
          // double announcements from VO when `focusRegionOnTextChange` is true
          aria-hidden={toggle ? undefined : 'true'}
          aria-live={focusRegionOnTextChange ? 'off' : ariaLive}
```

Reply:
That was my finding too, needing both. I got the double announcement smoothed out with the `aria-live` conditional, but then the strange behavior on toggle led to the `aria-hidden` being needed this morning. I'll go ahead and commit your suggested change.